Content.Server/StationEvents/Events/DiseaseOutbreak.cs

```diff
@@ -61,15 +64,24 @@ public override void Startup()
             return;
 
         var diseaseSystem = EntitySystem.Get<DiseaseSystem>();
-        /// Now we give it to people in the list of living disease carriers earlier
+        // Now we give it to people in the list of living disease carriers earlier
+        var _stationSystem = EntitySystem.Get<StationSystem>();
         foreach (var target in aliveList)
         {
             if (toInfect-- == 0)
                 break;
 
             diseaseSystem.TryAddDisease(target.Owner, disease, target);
+
+            var station = _stationSystem.GetOwningStation(target.Owner);
+            if(station == null) continue;
+            stationsToNotify.Add((EntityUid) station);
```

Comment:
wut

Reply:
it only notifies relevant stations (for multi-station) so if the player isn't on a station it doesn't do it

Reply:
I was referring to the cast, normally just .Value is used but I guess it's okay.